src/Http/Routing/src/RouteValuesAddress.cs

```diff
@@ -11,7 +11,7 @@ public class RouteValuesAddress
         /// <summary>
         /// Gets or sets the route name.
         /// </summary>
-        public string RouteName { get; set; }
+        public string? RouteName { get; set; }
```

Comment:
Why is the compiler ok with `ExplicitValues` not being nullable? It isn't set to a value on creation.

Reply:
It's not. The type didn't have nullable enabled. Turning it off unraveled more nullable things in routing.